CMakeLists.txt

```diff
@@ -158,10 +158,12 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/orionTypes)
   ADD_SUBDIRECTORY(src/lib/ngsi)
   ADD_SUBDIRECTORY(src/lib/serviceRoutines)
+  ADD_SUBDIRECTORY(src/lib/serviceRoutinesV2)
```

Comment:
We were discussing on using lib/serviceRoutines/v2 for v2 service routines and (to keep thing homogenoeus) move existing lib/serviceRoutines to lib/serviceRoutines/v1...